cmd/crawl.go

```diff
@@ -178,8 +178,8 @@ A max depth of 0 will open up an archive on the filesystem but not any nested ar
 	cmd.Flags().IntVar(&archivesCrawledPerSecond, "archives-per-second-rate-limit", 0, `The maximum number of archives to scan per second. 0 for unlimited.`)
 	cmd.Flags().BoolVar(&enableObfuscationDetection, "enable-obfuscation-detection", true, `Enable applying partial bytecode matching to Jars that appear to be obfuscated.`)
 	cmd.Flags().BoolVar(&enablePartialMatchingOnAllClasses, "enable-partial-matching-on-all-classes", false, `Enable partial bytecode matching to all class files found.`)
-	cmd.Flags().Uint32Var(&obfuscatedClassNameAverageLength, "maximum-average-obfuscated-class-name-length", 3, `The maximum average class name length for classes within a Jar to be considered obfuscated.`)
-	cmd.Flags().Uint32Var(&obfuscatedPackageNameAverageLength, "maximum-average-obfuscated-package-name-length", 3, `The maximum average package name length for packages within a Jar to be considered obfuscated.`)
+	cmd.Flags().IntVar(&obfuscatedClassNameAverageLength, "maximum-average-obfuscated-class-name-length", 3, `The maximum class name length for a class to be considered obfuscated.`)
```

Comment:
nit: Any reason to change this to int?
We get negative value checking from the flag framework for free if it remains as a uint based number.

Reply:
Previously it was float32 in the place where this value is eventually used, which the uint32 could be converted in to without any possibility of error. Now the eventual comparison is against an int, and you can't do a direct int to uint comparison, so we'd have to convert the uint to an int and potentially be too large at that point.

If somebody supplies a negative value it would be the same as supplying 0, we'd never consider anything obfuscated.